compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -272,10 +272,10 @@ public BType checkType(Location pos,
         return symTable.semanticError;
     }
 
-    public boolean isLax(BType type) {
+    public boolean isLaxFieldAccessAllowed(BType type) {
```

Comment:
Please add positive test cases too.

Reply:
The refactored changes are still covered with `XMLAccessTest` and `XMLAttributeAccessTest`, shall we add them elsewhere too?